server/Arcadia.Assistant.Organization/OrganizationActor.cs

```diff
@@ -52,6 +51,10 @@ protected override void OnReceive(object message)
                     this.employeesActor.Forward(query);
                     break;
 
+                case GetNextVacationRequestApprover msg:
```

Comment:
in what scenario is that used?

Reply:
https://github.com/SoftwareCountry/arcadian-assistant/pull/299/files/077466f3825bf7a890237136345194404833c50b#diff-89e87f12de8409707af9e31c732e4cedR110

Reply:
Legacy :)

Reply:
Removed.